app/core/initialize.coco.coffee

```diff
@@ -117,6 +118,16 @@ setUpMoment = ->
   me.on 'change:preferredLanguage', (me) ->
     setMomentLanguage me.get('preferredLanguage', true)
 
+setUpTv4 = ->
+  forms = require 'core/forms'
+  tv4.addFormat({
```

Comment:
There are a couple places that that email address / phone format things are added, like [here](https://github.com/codecombat/codecombat/blob/master/app/views/core/CreateAccountModal/BasicInfoView.coco.coffee#L165-L165) and [here](https://github.com/codecombat/codecombat/blob/master/app/views/teachers/CreateTeacherAccountView.ozar.coffee#L186-L186). Can we move all of these, either to here or to [forms.coffee](https://github.com/codecombat/codecombat/blob/master/app/core/forms.coffee)?